js/calculate.js

```diff
@@ -0,0 +1,127 @@
+const operate = require('./operate');
```

Comment:
can you refactor this to use import and not require? require would be harder to work with on the broswer.

Reply:
Done